src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FIgureSupplier figureSupplier = new FIgureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
```

Comment:
You are creating a new 'FigureSupplier' instance twice in the loops. According to the checklist, you should create a 'FigureSupplier' instance only once, before the loop starts.